.travis.yml

```diff
@@ -93,6 +90,16 @@ jobs:
     script:
     - make test CHARTS=reactive-sandbox
 
+  - name: Console Whitesource Report
+    # Only deploy for pushes to master (and if manually invoked)
+    #if: type IN (push, api)
```

Comment:
uncomment?

Reply:
Yes, will do. Wanted to test that it worked on the PR.